portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,37 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import java.util.List;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> comments;
+
+  @Override
+  public void init() {
+    comments = new ArrayList<String>();
+    comments.add("This is the first comment.");
+    comments.add("The second comment is right here.");
+    comments.add("The third comment has arrived.");
+  }
+
+   /**
+   * Converts a ServerStats instance into a JSON string using the Gson library.
```

Comment:
ServerStats instance? Was this maybe a copy-paste error?

Reply:
You're right, that was a copy / paste error; I've fixed this as well.